patches/0002-Fix-and-update-byte-count.patch

```diff
@@ -0,0 +1,46 @@
+From 9fa329dd092cfd19274bb6a3e6944ca8fa5095f0 Mon Sep 17 00:00:00 2001
+From: Eduard Mihailescu <mihailescu.eduard@gmail.com>
+Date: Sun, 26 Mar 2023 11:16:24 +0300
+Subject: [PATCH] Fix and update byte count Modify the way of doing division
```

Comment:
This requires a little bit of rework, especially if you try to upstream it. See https://github.com/unikraft/lib-musl

Reply:
Besides combining the two commits into a single one and removing trailing white spaces should I do anything else?

Reply:
My bad, I should had provided more accurate information. I was referring to the subject line, that seems to have (accidentally?) concatenated the description of the changes.